cli/src/main/java/com/devonfw/tools/ide/tool/GlobalToolCommandlet.java

```diff
@@ -70,6 +72,19 @@ protected boolean runWithPackageManager(boolean silent, List<PackageManagerComma
     return false; // None of the package manager commands were successful
   }
 
+  private void logPackageManagerCommands(List<PackageManagerCommand> pmCommands) {
+    StringBuilder commandLog = new StringBuilder("We need to run the following privileged command(s):\n");
```

Comment:
you do not need a `StringBuilder` here. Simply call `info` method 3 times and remove the `\n`s will do.